### PR TITLE
update example inventory files with new pods-per-core flag

### PR DESCRIPTION
**Pull request by jeremyeder**
and delete aep inventory files

### CODE REVIEW

**Comment by detiber** [comment]
@sdodson @adellape probably need to get this change into the docs as well.

lgtm

**Comment by jeremyeder** [comment]
@timothysc is gonna send a PR against openshift-docs. And we have this for
the actual implementation:

https://bugzilla.redhat.com/show_bug.cgi?id=1371309

On Aug 29, 2016 22:28, "Jason DeTiberus" <notifications@github.com> wrote:

> <https://github.com/sdodson>

> @sdodson <https://github.com/sdodson> @adellape
<https://github.com/adellape> probably need to get this change into the
docs as well.
>
> lgtm
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
<https://github.com/openshift/openshift-ansible/pull/2381#issuecomment-243315341>,
or mute the thread

File: inventory/byo/hosts.ose.example [hunk 1]
```diff
@@ -465,7 +465,7 @@ openshift_master_identity_providers=[{'name': 'htpasswd_auth', 'login': 'true',
 #openshift_dns_ip=172.30.0.1
 
 # Configure node kubelet arguments
-#openshift_node_kubelet_args={'max-pods': ['110'], 'image-gc-high-threshold': ['90'], 'image-gc-low-threshold': ['80']}
+#openshift_node_kubelet_args={'pods-per-core': ['10'], 'max-pods': ['250'], 'image-gc-high-threshold': ['90'], 'image-gc-low-threshold': ['80']}
```
Comment (timothysc) [inline inventory/byo/hosts.ose.example]: Did you verify that when you apply it reflects in the node status?  Openshift has been known to filter/ignore arguments.

Comment (detiber) [inline inventory/byo/hosts.ose.example]: @sdodson sounds like we need to make sure there is a QE test case for this.

**Comment by jeremyeder** [comment]
Using the line from the commit and running through a fresh install, you end up with:

```
# grep -A8 kubeletArguments /etc/origin/node/node-config.yaml 
kubeletArguments: 
  image-gc-high-threshold:
  - '90'
  image-gc-low-threshold:
  - '80'
  max-pods:
  - '250'
  pods-per-core:
  - '10'
```

This is a 4-vCPU VM on Amazon, and the capacity shows up as expected based on the above kubeletArguments

```(pods-per-core=10, 4vCPUs, capacity:pods should be 40)```

And it is:

```
Capacity:
 alpha.kubernetes.io/nvidia-gpu:        0
 cpu:                                   4
 memory:                                16005268Ki
 pods:                                  40
Allocatable:
 alpha.kubernetes.io/nvidia-gpu:        0
 cpu:                                   4
 memory:                                16005268Ki
 pods:                                  40
```

**Comment by timothysc** [comment]
lgtm.

**Pull request merged**